doc/files.md

```diff
@@ -26,40 +26,42 @@ Linux    | `$HOME/.bitcoin/`
 macOS    | `$HOME/Library/Application Support/Bitcoin/`
 Windows  | `%APPDATA%\Bitcoin\` <sup>[\[1\]](#note1)</sup>
 
-2. The non-default data directory path can be specified by `-datadir` option.
+2. The data directory path can be specified with `-datadir` option.
 
 3. All content of the data directory, except for `bitcoin.conf` file, is chain-specific. This means the actual data directory paths for non-mainnet cases differ:
 
-Chain option        | Data directory path
---------------------|--------------------
-no option (mainnet) | *path_to_datadir*`/`
-`-testnet`          | *path_to_datadir*`/testnet3/`
-`-regtest`          | *path_to_datadir*`/regtest/`
+Chain option                   | Data directory path
+-------------------------------|------------------------------
+`-chain=main` (by default)     | *path_to_datadir*`/`
+`-chain=signet` or `-signet`   | *path_to_datadir*`/signet/`
```

Comment:
generally, the chain order in the codebase is main/test/signet/regtest

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/20076#issuecomment-703294876).